src/pm-conversations/PmConversationList.js

```diff
@@ -24,17 +31,33 @@ type Props = $ReadOnly<{|
 /**
  * A list describing all PM conversations.
  * */
-export default class PmConversationList extends PureComponent<Props> {
+class PmConversationList extends PureComponent<Props> {
   handleUserNarrow = (email: string) => {
     this.props.dispatch(doNarrow(privateNarrow(email)));
   };
 
   handleGroupNarrow = (users: UserOrBot[]) => {
-    this.props.dispatch(doNarrow(groupNarrow(users.map(s => s.email))));
+    const { dispatch, ownUser } = this.props;
+    const emails = users.filter(u => u.user_id !== ownUser.user_id).map(u => u.email);
+    dispatch(doNarrow(groupNarrow(emails)));
+  };
```

Comment:
Ah, I see. This change belongs in the earlier commit where we changed what set of users gets passed here.

Reply:
– oh, there it is. I must have squashed the fix into the wrong commit. :sweat: